sps30.cpp

```diff
@@ -107,11 +108,21 @@ s16 sps30_read_data_ready(u16 *data_ready) {
 s16 sps30_read_measurement(struct sps30_measurement *measurement) {
     s16 ret;
     u16 idx;
+
+
+#ifdef SPS30_USE_I2C_32_BYTE_LIMIT
+    union {
+        u16 value_u16[2];
+        u32 u;
+        f32 f;
+    } val, data[4];
```

Comment:
The only difference seems `data[4]` vs `data[10]` - use a `#define` instead of duplicating the union definition?

Reply:
Will do